config/test-integration-session-3.json

```diff
@@ -0,0 +1,6 @@
+{
+  "storageProfile": "testIntegration3Profile",
```

Comment:
This config is un-necessary, we already have a dedicated profile for `test-integration`

Reply:
This is for future tests involving groups of >2 members.

Reply:
Ye but all this config does is setup a storage profile, which we do automatically based on the instance and environment.